src/ApplicationAspectKernel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace KickAss\Commerce;
+
+use Go\Core\AspectKernel;
+use Go\Core\AspectContainer;
+
+/**
+ * Application Aspect Kernel
+ */
+class ApplicationAspectKernel extends AspectKernel
+{
+
+    /**
+     * Configure an AspectContainer with advisors, aspects and pointcuts
+     *
+     * @param AspectContainer $container
+     * @return void
+     */
+    protected function configureAop(AspectContainer $container)
```

Comment:
why is this protected? can we either have public or private in our world? I would love you more than red wine if we could work in a world of only public and privet methods!